charts/common/templates/_annotation.tpl

```diff
@@ -0,0 +1,14 @@
+{{- define "accelleran.common.annotations" -}}
+{{- $ := get . "top" | required "The top context needs to be provided to common annotations" -}}
+{{- $values := get . "values" | default $.Values -}}
+
+{{- $annotations := get . "annotations" -}}
+
+{{- if $annotations -}}
+{{- $annotations | toYaml }}
+{{- else -}}
+{{- with $values.annotations -}}
+{{ toYaml . }}
+{{- end -}}
```

Comment:
Is that an explicit difference between `{{- $annotations | toYaml -}}` and `{{- with $annotations -}}{ toYaml . }}{{- end -}}`?

Reply:
No, but opening another context is unnecessary with the if statement above it.